addons/website_sale/models/account_move.py

```diff
@@ -18,3 +18,14 @@ def preview_invoice(self):
             # URL should always be relative, safety check
             action['url'] = f'/@{action["url"]}'
         return action
+
+    @api.depends('date')
```

Comment:
```suggestion
    @api.depends('partner_id')  # Dummy depends to trigger compute, will be dropped in master
```
`date` isn't a bad idea, but to go to the easiest solution, I'd say that we could keep the `partner_id`, which was already a dependency of the previous field anyway.